.devcontainer/Dockerfile

```diff
@@ -17,14 +19,18 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- \
     --default-toolchain ${NIGHTLY_VERSION} -y \
     && $HOME/.cargo/bin/rustup component add rust-src --toolchain ${NIGHTLY_VERSION} \
     && $HOME/.cargo/bin/rustup target add riscv32i-unknown-none-elf \
-    && $HOME/.cargo/bin/cargo install cargo-generate cargo-espflash espmonitor bindgen ldproxy
-ENV ESP_BOARD=esp32c3
-ENV ESP_IDF_VERSION=release/v4.4
+    && $HOME/.cargo/bin/cargo install cargo-espflash espmonitor ldproxy
 RUN mkdir -p .espressif/frameworks/ \
     && git clone --branch ${ESP_IDF_VERSION} -q --depth 1 --shallow-submodules \
     --recursive https://github.com/espressif/esp-idf.git \
     .espressif/frameworks/esp-idf-v4.4 \
+    && python3 .espressif/frameworks/esp-idf-v4.4/tools/idf_tools.py install cmake \
     && .espressif/frameworks/esp-idf-v4.4/install.sh ${ESP_BOARD} \
     && rm -rf .espressif/dist \
-    && rm -rf .espressif/frameworks/esp-idf-v4.4/docs
+    && rm -rf .espressif/frameworks/esp-idf-v4.4/docs \
+    && rm -rf .espressif/frameworks/esp-idf-v4.4/examples \
```

Comment:
Cleaning even more unnecessary folders is good :+1:

Reply:
Hello! Thanks for your feedback! I will be addressing the other comments soon. Those changes on the Dockerfile are in another PR: https://github.com/ferrous-systems/espressif-trainings/pull/69, maybe we should first merge those changes since it reduces build time (by properly activating esp-idf environment) and the Dockerfile is more polished!